packages/patternfly-4/react-inline-edit-extension/src/components/InlineEdit/utils/formatters/inlineEditFormatterFactory.d.ts

```diff
@@ -3,8 +3,8 @@ import { ReactNode } from 'react';
 export interface AdditionalData {
   columnIndex: number;
   rowIndex: number;
-  column: Object;
-  rowData: Object;
+  column: Record<string, any>;
+  rowData: Record<string, any>;
```

Comment:
This might be the right thing to do. But I am concerned again that we are changing type values in an exported interface.

Reply:
Thanks - reverted this and disabled linter error for these lines